### ISSUE TITLE
DSV2 support for redshift

### ISSUE THREAD

**Issue by karuppayya** [spark-redshift-community/spark-redshift#69]
With Spark 3.0(to be released soon) we will have read support for for DSv2. I am planning to use the DSv2 api for Redshift connector. Wanted to check if the community would be interested in this?

**Comment by smoy** [comment]
yes yes yes. that will be amazing. have been waiting for DSv2 to stabilize the api, which DSv2 doc you are using, i like to take a look as well. Cheers.

**Comment by karuppayya** [comment]
@smoy I have created an PR. Please take a look.
https://github.com/spark-redshift-community/spark-redshift/pull/70
